ui/app/AppLayouts/Wallet/views/CollectiblesView.qml

```diff
@@ -92,7 +92,7 @@ ColumnLayout {
                 clear()
                 if (d.isLoading) {
                     for (let i = 0; i < 10; i++) {
-                        append({ isLoading: true })
+                        append({ isLoading: true, name: "Loading dummy" })
```

Comment:
Could be better to have nice placeholders just in case it will reach the UI at some point 😄 
```suggestion
                        append({ isLoading: true, name: "Loading collectible..." })
```

Reply:
And if it does ;)
```suggestion
                        append({ isLoading: true, name: qsTr("Loading collectible...") })
```